OWNERS_ALIASES

```diff
@@ -101,6 +101,7 @@ aliases:
     - mm4tt
     - shyamjvs
     - wojtek-t
+    - marseel
```

Comment:
Let's remove @mm4tt and sort alphabetically :)

Reply:
Done